dataplane/src/main.rs

```diff
@@ -80,7 +81,7 @@ fn main() {
         }
         other => {
             error!("Unknown driver '{other}'. Aborting...");
-            std::process::exit(0);
+            std::process::exit(0); // TODO - same as above
```

Comment:
again, prefer to call `error!` and then `panic!`

Reply:
switched to `panic!`